src/tls.c

```diff
@@ -12137,16 +12137,25 @@ int TLSX_PopulateExtensions(WOLFSSL* ssl, byte isServer)
             if (usingPSK)
             #endif
             {
-                byte modes;
+                byte modes = 0;
 
                 (void)usingPSK;
                 /* Pre-shared key modes: mandatory extension for resumption. */
-                modes = 1 << PSK_KE;
+            #ifdef HAVE_SUPPORTED_CURVES
+fprintf(stderr, "ONLY PSK DHE KE: %d\n", ssl->options.onlyPskDheKe);
```

Comment:
Did you mean for this to get included in the PR?

Reply:
Removed